fastlane/helper/semantic_release_helper.rb

```diff
@@ -0,0 +1,97 @@
+require 'fastlane_core/ui/ui'
+
+module Fastlane
+  UI = FastlaneCore::UI unless Fastlane.const_defined?("UI")
+
+  module Helper
+    class SemanticReleaseHelper
+      # class methods that you define here become available in your action
+      # as `Helper::SemanticReleaseHelper.your_method`
+      #
+      def self.git_log(pretty, start)
+        command = "git log --pretty='#{pretty}' --reverse #{start}..HEAD"
+        Actions.sh(command, log: false).chomp
+      end
+
+      def self.parse_commit(params)
+        commit_subject = params[:commit_subject].strip
+        commit_body = params[:commit_body]
+        releases = params[:releases]
+        codepush_friendly = params[:codepush_friendly]
+        pattern = /^(docs|fix|feat|chore|style|refactor|perf|test)(\((.*)\))?(!?)\: (.*)/
```

Comment:
1. As above, consider multiline commented regexes.
1. Why does the colon need to be escaped here?
1. The capture of the scope probably wants to be non-greedy, or else it will fail on commits like `feat(lang): add Chinese (simplified) language`
1. Probably want a `\s*` rather than a hardcoded single space between the colon and the message capture